google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsTracer.java

```diff
@@ -239,26 +249,44 @@ private void recordOperationCompletion(@Nullable Throwable status) {
       return;
     }
     operationTimer.stop();
+
+    boolean isStreaming = operationType == OperationType.ServerStreaming;
+    String statusStr = Util.extractStatus(status);
+
+    Attributes attributes =
```

Comment:
We can change all the attributes to map in this class so this class is agnostic of OpenTelemetry.